src/inputs/select.scss

```diff
@@ -3,113 +3,75 @@
 @import '../style/index';
 @import '../menu/index';
 
-$iui-after-border: 5px solid;
+@mixin iui-select-button {
+  @include iui-focus($offset: -2px, $thickness: 2px);
```

Comment:
Maybe we should move focus styling to the parent and use `focus-within` so that it's consistent with combobox

Reply:
adding focus to parent could be difficult as `iui-select-button` have background color and border which cover `iui-input-with-icon` borders and outline.

Reply:
`focus-within` on select-button doesn't do anything because the menu will not be inside the button.

You can target `:focus-within` on the parent and apply the actual styles to the child

Reply:
moved focus-within to input with icon

Reply:
I think it wont work with status inputs (or maybe it will show double outline). You might want to link this repo to React, make the necessary changes, and see what happens.

My guess is that you would need to remove `iui-focus` mixin in `select.scss` and remove `:focus` styling for `iui-select-button` in `labeled-inputs.scss` and change it to just outline-color
![image](https://user-images.githubusercontent.com/9084735/152201427-7f647833-6470-4519-81d0-8bcf0bc15123.png)